auto_nag/bzcleaner.py

```diff
@@ -347,9 +347,25 @@ def set_needinfo(self, dryrun):
                 res[bugid] = data
         return res
 
-    def has_individual_autofix(self):
-        """Check if the autofix is the same for all bugs or different for each bug"""
-        return False
+    def has_individual_autofix(self, changes):
+        # check if we have a dictionary with bug numbers as keys
+        # return True if all the keys are bug number
+        # (which means that each bug has its own autofix)
+        # and clean the changes (replace bugid as int by bugid as str)
+        res = {}
+        individual = True
+        for bugid, data in changes.items():
+            if isinstance(bugid, six.integer_types):
+                res[bugid] = data
+            else:
+                if individual and not bugid.isdigit():
+                    individual = False
```

Comment:
You could do something like `return all(isinstance(bugid, six.integer_types) or bugid.isdigit() for bugid in changes.keys())`